website/docs/docs/deploy/job-commands.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Job commands"
+id: "job-commands"
+description: "How to use dbt commands to set tasks for your dbt Cloud jobs."
+---
+
+A dbt Cloud production job allows you to set up a system to run a dbt job and job commands on a schedule, rather than running dbt commands manually from the command line or [IDE](/docs/get-started/develop-in-the-cloud). A job consists of multiple commands, and their combined outcomes determine the job's run status, which can be successful, canceled, or failed. During a job run, each command outcome can either succeed or fail under the **Run steps** section.
```

Comment:
Can we add a little clarity around that last sentence? "fail or run under the **Run steps** section". Does that mean it's a log of why it failed or is it a status page?

Reply:
rephrased:

>A dbt Cloud production job allows you to set up a system to run a dbt job and job commands on a schedule, rather than running dbt commands manually from the command line or [IDE](/docs/get-started/develop-in-the-cloud). A job consists of commands that are "chained" together and executed as run steps. Each run step can succeed or fail, which may determine the job's run status (Success, Cancel, or Error).